nx-X11/programs/Xserver/hw/nxagent/Extensions.c

```diff
@@ -114,13 +163,64 @@ void nxagentInitRandRExtension(ScreenPtr pScreen)
   #endif
 }
 
-void
-RRResetProc (ExtensionEntry *extEntry)
+#if RANDR_14_INTERFACE
+static Bool
+nxagentRandRReplaceScanoutPixmap(DrawablePtr pDrawable,
+				 PixmapPtr pPixmap,
+				 Bool enable)
 {
-  fprintf(stderr, "RANDR going down - NX version\n");
+  /* FALSE means: not supported */
+  fprintf(stderr, "nxagentRandRReplaceScanoutPximap: NX's RANDR does not support scan-out pixmaps.\n");
```

Comment:
typo in function name in fprintf().

Also, do we really need that message? It should be sufficient for the calling party to get the return code.